packages.md

```diff
@@ -7,36 +7,44 @@ id: packages
 <div class="packages">
   <div class="container" markdown="1">
 
-## Desktop version
-
-{% include download_button.html %}
-
-This is the latest version of the desktop version of OpenRocket. It contains
-the latest and greatest features, and runs on any system supporting Java.
-
-OpenRocket can be started in most graphical environments (including Windows) by
-double-clicking the package icon. No installation is required.
-
-From the command line OpenRocket can be started by `java -jar OpenRocket-15.03.jar`
-
-
-## Android version
-
-The Android version allows opening OpenRocket files, viewing simulations and motors. Later versions will allow running simulations as well.
-
-The Android version is thanks to the fantastic work by Kevin Ruland.
-
-Installation:
-
- 1. Go to Settings → Applications and check Unknown sources.
- 1. Download [OpenRocket-Android-13.11.apk](https://github.com/openrocket/openrocket/releases/download/release-android-13.11/OpenRocket-Android-13.11.apk) on your device and accept installation.
-
+OpenRocket is available as packages for Linux, macOS and Windows.  A .jar file
+is also available, as is all of the source code.
+
+For most users, we **strongly** recommend using the package
+appropriate for your operating system.  These packages contain all of
+the needed dependencies, including the correct version of Java (Java 8).
+
+## macOS
+The macOS package is a .dmg file.  Download it, and drag it into your
+Applications folder.  
+<a class="btn btn-primary btn-lg" href="https://github.com/openrocket/openrocket/releases/download/release-{{site.current_version}}/OpenRocket-{{site.current_version}}.dmg" role="button">Download OpenRocket-{{site.current_version}}.dmg</a>
+
+## Linux
+The Linux package is an AppImage.  Download it, make it executable  
+`chmod +x OpenRocket-{{site.current_version}}.AppImage`  
+and run it  
+`./OpenRocket-{{site.current_version}}.AppImage`  
+<a class="btn btn-primary btn-lg" href="https://github.com/openrocket/openrocket/releases/download/release-{{site.current_version}}/OpenRocket-{{site.current_version}}.AppImage" role="button">Download OpenRocket-{{site.current_version}}.AppImage</a>
```

Comment:
As an FYI, users will also be able to use the snap installation from the snap store on linux. Its available now, but it delivers the unstable branch so I think we should hold off on mentioning it here. Just a note for future self more than anything

Reply:
Not quite sure why the lines of code quoted in your comment -- is there a suggested change in there?

Right, I don't think we want to announce the snap on our main download page until we're in beta.

Reply:
uh, its just where I clicked in relation to the linux context. sorry for the confusion